content/docs/start/bitbucket.md

```diff
@@ -0,0 +1,71 @@
+# Get Started with CML on Bitbucket
+
+Here, we'll walk through a tutorial to start using CML. For simplicity, we'll
+show the demo in Bitbucket Pipelines, but instructions are pretty similar for
+all the supported CI systems.
+
+1. Fork our
+   [example project repository](https://bitbucket.org/iterative-ai/example-cml).
+
+   ![](/img/bitbucket_fork_cml_project.png)
+
+   The following steps can all be done in the Bitbucket browser interface.
+   However, to follow along the commands, we recommend cloning your fork to your
+   local workstation:
+
+   ```bash
+   git clone https://bitbucket.org/<your-username>/example-cml
+   ```
+
+2. To create a CML workflow, copy the following into a new file,
```

Comment:
```suggestion
2. To create a CML workflow, copy the following into a new file on your main branch,
```
lets either add this small caveat or add instructions to add the pipeline file first to the main branch.

In my testing, it doesn't work if its part of the pr which following these steps explicitly it is.